msal-dotnet-articles/how-to/token-cache-serialization.md

```diff
@@ -16,11 +16,19 @@ ms.custom: devx-track-csharp, aaddev, has-adal-ref, devx-track-dotnet
 #Customer intent: As an application developer, I want to learn about token cache serialization so I can have fine-grained control of the proxy.
 ---
 
-# Token cache serialization in MSAL.NET
+# Token cache serialization
 
 After Microsoft Authentication Library (MSAL) [acquires a token](/azure/active-directory/develop/msal-acquire-cache-tokens), it caches that token. Public client applications (desktop and mobile apps) should try to get a token from the cache before acquiring a token by another method. Acquisition methods on confidential client applications manage the cache themselves. This article discusses default and custom serialization of the token cache in MSAL.NET.
 
-## Quick summary
+## Token cache types
```

Comment:
I would put this section at the bottom of the page. The token cache types are ideally abstracted away from the developers when either the static cache or Identity Web caches are used, and only matters when custom caching is used, which is not recommended anyway.